### ISSUE TITLE
Possible incorrect 'name' when inviting an existing group member

### ISSUE THREAD

**Issue by ryanjohndias** [kin-business/kin.api.public#4]
If this is not a bug, please consider it a request :)

When fetching a `UserInvite` via `userInviteByPublicId`, the `name` field contains the name of the invitee, instead of the group name. I'm creating the invite via `groupInviteExistingMember`

`UserInvite` doesn't have any other references to a group; We need the name of the group to display to the user in the invite acceptance UI

**Comment by ryanjohndias** [comment]
![use case](https://user-images.githubusercontent.com/6240969/45359612-555ecf00-b5cd-11e8-86b8-421f79abb0cb.png)

**Comment by rolfwessels** [comment]
Makes sence. I will add it and let you know.

**Comment by rolfwessels** [comment]
Ok I have added the follwing. @ryanjohndias  Will that work for you ?

```
 userInvites {
    invitingUser {id, name }
    inviteToGroup {id, name }
  }
```

It is not rolled out yet.. just asking

**Comment by ryanjohndias** [comment]
Perfect - thanks 🙏

**Comment by rolfwessels** [comment]
fixed in released 0.0.2.49
